sklearn/inspection/_ice.py

```diff
@@ -0,0 +1,491 @@
+"""Individual Conditional Expectation (ICE) plots for regression and
+classification models."""
+
+# Authors: Peter Prettenhofer
```

Comment:
do you really think this file was such a broad collaboration?

Reply:
Before @glemaitre suggested using a _base.py I had moved/extracted common functions into _ice.py. Just wanted to give credit to the original authors of those code segments. I should have moved the credit to _base.py.